### PR TITLE
Release Notes - September 17, 2024

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

My PR adds release notes for the September 17, 2024 dashboard release.

### Related PRs, issues, or features (optional)

NONE

### Feature release date (optional)

17 September 2024

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_help/release_notes/2024/9_17_24.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## WhatsApp response messages
+
+{% multi_lang_include release_type.md release="General availability" %}
+
+You can use [response messages]({{site.baseurl}}/user_guide/message_building_by_channel/whatsapp/whatsapp_campaign/create#response-messages) to reply to inbound WhatsApp messages from your users. These messages are built in-app on Braze during your composition experience and can be edited at any time. You can use Liquid to match the response message language to the appropriate users.
+
+## Canvas templates
+
+{% multi_lang_include release_type.md release="General availability" %}
```
[the remark this answers is not in the document]
Comment (rachel-feinberg) [reply _docs/_help/release_notes/2024/9_17_24.md:34]: It's going to be GA today, but the PR that pulls it into the main nav doesn't deploy until today 🫡

[the remark this answers is not in the document]
Comment (josh-mccrowell-braze) [reply _docs/_help/release_notes/2024/9_17_24.md:34]: Oh wow. My info is outdated. Good looking out!

File: _docs/_help/release_notes/2024/9_17_24.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+You can view the number of updates to your [Canvases]({{site.baseurl}}/user_guide/engagement_tools/canvas/testing_canvases/measuring_and_testing_with_canvas_analytics/#changes-since-last-viewed), campaigns, and [segments]({{site.baseurl}}/user_guide/engagement_tools/segments/managing_segments/#changes-since-last-viewed) by other members of your team by referring to the *Changes Since Last Viewed* metric on the respective overview pages (such as the overview page for an [email campaign]({{site.baseurl}}/user_guide/message_building_by_channel/email/reporting_and_analytics/email_reporting#changes-since-last-viewed)). 
+
+### Troubleshooting webhook and Connected Content requests 
+
+[This article]({{site.baseurl}}/help/help_articles/api/webhook_connected_content_errors) covers how to troubleshoot webhook and Connected Content error codes, including what the errors are and steps to resolve them.
+
+## New Braze partnerships
+
+### Inbox Monster
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2024/9_17_24.md:79]:
```suggestion
### Inbox Monster - Analytics
```

File: _docs/_help/release_notes/2024/9_17_24.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+[Inbox Monster]({{site.baseurl}}/partners/data_and_infrastructure_agility/analytics/inbox_monster/) is an inbox signals platform that helps enterprise brands land every send. It's an integrated suite of solutions for deliverability, creative rendering, and SMS monitoring, that empowers modern customer relationship managment (CRM) teams and ends the sending scaries.
+
+### SessionM
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2024/9_17_24.md:83]:
```suggestion
### SessionM - Loyalty
```

**Review by lydia-xie — APPROVED** [review]
lgtm! left 2 small edits for the partner headings

**Pull request merged**